owl/EASE-PROC.owl

```diff
@@ -356,32 +309,6 @@ Note, most of the physical Entities we are concerned with are actually arrangeme
     
 
 
-    <!-- http://www.ease-crc.org/ont/EASE-PROC.owl#EffectorContact -->
-
-    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-PROC.owl#EffectorContact">
```

Comment:
This should stay.

Reply:
Why? I do not think this is very useful.